Get dlcs new way

onadrog opened the pull request:
I found a new way to get dlcs using the 'old way' i think that's will be better since some `stdout` memory problems have been discovered.

flavioislima commented on electron/legendary_utils/library.ts:
```diff
@@ -131,14 +140,21 @@ export async function getLegendaryConfig(file: string) {
-
+            : {} 
+          
+          const dlc = () => {
```
I guess this function could be simplified if you use every instead of map.
For instance:
`const dlc = () => dlcs.every(dlc => dlc === app_name)`

flavioislima commented on electron/legendary_utils/library.ts:
```diff
@@ -105,8 +107,15 @@ export async function getLegendaryConfig(file: string) {
             customAttributes: { CloudSaveFolder, FolderName },
           } = metadata
+          dlcItemList ?
+            dlcItemList.map((v: { releaseInfo: any[] }) => {
```
Since the map function is not returning anything, you could use a forEach instead.
Also changing the check condition to be a normal if/else would be better for readability I guess.

flavioislima reviewed (commented):
Liked the idea and looks good in general compared with the old function. just left a few comments.

Also, if you think we won't use the old function anymore, you can then remove it from the file. 😃

onadrog commented:
yes, I'm working on it + adding system requirement ( disk usage, graphics cards etc...), I'll commit once finished

flavioislima commented:
> yes, I'm working on it + adding system requirement ( disk usage, graphics cards etc...), I'll commit once finished

nice. Let me know when you're ready :D

adityaruplaha commented:
> yes, I'm working on it + adding system requirement ( disk usage, graphics cards etc...), I'll commit once finished

Fixes #207 if implemented.

onadrog commented:
@flavioislima @D3SOX @entaromia @skowr @lol28 
can you pls give the translations for: 

```Javascript
 "requirements": "System Requirements"

    "minimum": "Minimum",
    "recommended": "Recommended"

```
If I forgot someone pls tag them.
Thx

entaromia commented:
For Turkish:

​"requirements"​: ​"Sistem Gereksinimleri"​

    ​"minimum"​: ​"Minimum"​,​
    ​"recommended"​: ​"Önerilen"

lol28 commented:
```
 "requirements": "Системные требования"

    "minimum": "Минимальные",
    "recommended": "Рекомендованные"
```

D3SOX commented:
German:
```json
 "requirements": "Systemanforderungen"
"minimum": "Minimal",
"recommended": "Empfohlen"
```

flavioislima commented:
Portuguese:
 "requirements": "Requisitos"
  "minimum": "Mínimo",
  "recommended": "Recomendado"

@onadrog I think for small translations, we could use Google translate since it will be crazy soon to ask so many people, and then send the PR link to the Discord for us to take a look.

skowr commented:
Hi guys
Polish translation:

"requirements": "Wymagania"
"minimum": "Minimalne",
"recommended": "Rekomendowane"

onadrog commented:
Some fixes (eg. games count now substract dlcs) changed the way we use `InfoBox` now we have to pass the key translation in `text` props so we can reuse this Component, added 'system requirements' table, added query params for images so we don't fetch oversized images.

flavioislima commented:
Looks great, man!
I'll take a look later today.

Can you add some screenshots on how game the information is being shown?

onadrog commented:
![screen1](https://user-images.githubusercontent.com/46002955/109147747-ab0b9f80-7765-11eb-8758-84caf0035683.png)
![screen2](https://user-images.githubusercontent.com/46002955/109147751-aba43600-7765-11eb-94eb-40df432321d4.png)
